### PR TITLE
Update GitOps naming to reflect new regions

### PR DESCRIPTION
**Pull request by TravisFallsMSFT**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
This PR changes all name references in files and paths that refer to westus2 --> westus3 "West Us2" --> "West Us3", and centralus to northcentralus as well as "Central Us" --> "North Central Us".  This change is to reflect the current deployment regions.  

## Does this introduce a breaking change

- [ ] YES
- [x] NO *** There are three PRs needed.

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/1013

### CODE REVIEW

File: monitoring/alerts/metricsAlerts/pre/asb-pre-centralus-AppEndpointDown.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    "name": "asb-pre-centralus-AppEndpointDown",
+    "name": "asb-pre-northcentralus-AppEndpointDown",
     "location": "global",
     "properties": {
         "description": "One or more application endpoints are down",
         "severity": 0,
         "enabled": true,
         "scopes": [
-            "/subscriptions/${Ngsa_Subscription_Guid}/resourceGroups/${Ngsa_Common_Services_RG}/providers/Microsoft.Network/applicationGateways/apw-aks-fykg5bqasutle-centralus"
+            "/subscriptions/${Ngsa_Subscription_Guid}/resourceGroups/${Ngsa_Common_Services_RG}/providers/Microsoft.Network/applicationGateways/apw-aks-fykg5bqasutle-northcentralus"
```
Comment (AAkindele) [inline monitoring/alerts/metricsAlerts/pre/asb-pre-centralus-AppEndpointDown.json:9]:
```suggestion
            "/subscriptions/${Ngsa_Subscription_Guid}/resourceGroups/${Ngsa_Common_Services_RG}/providers/Microsoft.Network/applicationGateways/apw-aks-ri3aov7twb4uy-northcentralus"
```

File: monitoring/alerts/metricsAlerts/pre/asb-pre-centralus-AppEndpointDown.json [hunk 2]
```diff
@@ -22,10 +22,10 @@
...
                             "operator": "Include",
                             "values": [
-                                "grafana-centralus-pre.cse.ms~grafana-centralus-pre-httpsettings",
-                                "ngsa-cosmos-centralus-pre.cse.ms~ngsa-cosmos-centralus-pre-httpsettings",
-                                "ngsa-java-centralus-pre.cse.ms~ngsa-java-centralus-pre-httpsettings",
-                                "ngsa-memory-centralus-pre.cse.ms~ngsa-memory-centralus-pre-httpsettings"
+                                "grafana-northcentralus-pre.cse.ms~grafana-northcentralus-pre-httpsettings",
+                                "ngsa-cosmos-northcentralus-pre.cse.ms~ngsa-cosmos-northcentralus-pre-httpsettings",
+                                "ngsa-java-northcentralus-pre.cse.ms~ngsa-java-northcentralus-pre-httpsettings",
+                                "ngsa-memory-northcentralus-pre.cse.ms~ngsa-memory-northcentralus-pre-httpsettings"
```
Comment (AAkindele) [inline monitoring/alerts/metricsAlerts/pre/asb-pre-centralus-AppEndpointDown.json:28]:
```suggestion
                                "grafana-northcentralus-pre.austinrdc.dev~grafana-northcentralus-pre-httpsettings",
                                "ngsa-cosmos-northcentralus-pre.austinrdc.dev~ngsa-cosmos-northcentralus-pre-httpsettings",
                                "ngsa-java-northcentralus-pre.austinrdc.dev~ngsa-java-northcentralus-pre-httpsettings",
                                "ngsa-memory-northcentralus-pre.austinrdc.dev~ngsa-memory-northcentralus-pre-httpsettings"
```

File: monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-    "name": "asb-pre-westus2-AppEndpointDown",
+    "name": "asb-pre-westus3-AppEndpointDown",
     "location": "global",
     "properties": {
         "description": "One or more application endpoints are down",
         "severity": 0,
         "enabled": true,
         "scopes": [
-            "/subscriptions/${Ngsa_Subscription_Guid}/resourceGroups/${Ngsa_Common_Services_RG}/providers/Microsoft.Network/applicationGateways/apw-aks-fykg5bqasutle-westus2"
+            "/subscriptions/${Ngsa_Subscription_Guid}/resourceGroups/${Ngsa_Common_Services_RG}/providers/Microsoft.Network/applicationGateways/apw-aks-fykg5bqasutle-westus3"
```
Comment (AAkindele) [inline monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json:9]:
```suggestion
            "/subscriptions/${Ngsa_Subscription_Guid}/resourceGroups/${Ngsa_Common_Services_RG}/providers/Microsoft.Network/applicationGateways/apw-aks-ri3aov7twb4uy-westus3"
```

File: monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json [hunk 4]
```diff
@@ -22,10 +22,10 @@
...
                             "operator": "Include",
                             "values": [
-                                "grafana-westus2-pre.cse.ms~grafana-westus2-pre-httpsettings",
-                                "ngsa-cosmos-westus2-pre.cse.ms~ngsa-cosmos-westus2-pre-httpsettings",
-                                "ngsa-java-westus2-pre.cse.ms~ngsa-java-westus2-pre-httpsettings",
-                                "ngsa-memory-westus2-pre.cse.ms~ngsa-memory-westus2-pre-httpsettings"
+                                "grafana-westus3-pre.cse.ms~grafana-westus3-pre-httpsettings",
+                                "ngsa-cosmos-westus3-pre.cse.ms~ngsa-cosmos-westus3-pre-httpsettings",
+                                "ngsa-java-westus3-pre.cse.ms~ngsa-java-westus3-pre-httpsettings",
+                                "ngsa-memory-westus3-pre.cse.ms~ngsa-memory-westus3-pre-httpsettings"
```
Comment (AAkindele) [inline monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json:28]:
```suggestion
                                "grafana-westus3-pre.austinrdc.dev~grafana-westus3-pre-httpsettings",
                                "ngsa-cosmos-westus3-pre.austinrdc.dev~ngsa-cosmos-westus3-pre-httpsettings",
                                "ngsa-java-westus3-pre.austinrdc.dev~ngsa-java-westus3-pre-httpsettings",
                                "ngsa-memory-westus3-pre.austinrdc.dev~ngsa-memory-westus3-pre-httpsettings"
```

**Review by AAkindele — COMMENTED** [review]
Found a few references to old resources in the `monitoring/alerts/metricsAlerts` json files. Probably worth a quick double check of the other files to see if the resource names need to be updated.

File: monitoring/alerts/metricsAlerts/dev/asb-dev-westus2-AppEndpointDown.json [hunk 5]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (AAkindele) [inline monitoring/alerts/metricsAlerts/dev/asb-dev-westus2-AppEndpointDown.json:1]: This file should be renamed this file to use westus3. The other central, and westus2 metricAlerts files should be renamed as well.

File: monitoring/alerts/metricsAlerts/dev/asb-dev-westus2-AppEndpointDown.json [hunk 6]
```diff
@@ -22,11 +22,11 @@
...
-                                "grafana-westus2-dev.cse.ms~grafana-westus2-dev-httpsettings",
-                                "ngsa-cosmos-westus2-dev.cse.ms~ngsa-cosmos-westus2-dev-httpsettings",
-                                "ngsa-java-westus2-dev.cse.ms~ngsa-java-westus2-dev-httpsettings",
-                                "ngsa-memory-westus2-dev.cse.ms~ngsa-memory-westus2-dev-httpsettings",
-                                "loderunner-westus2-dev.cse.ms~loderunner-westus2-dev-httpsettings"
+                                "grafana-westus3-dev.cse.ms~grafana-westus3-dev-httpsettings",
+                                "ngsa-cosmos-westus3-dev.cse.ms~ngsa-cosmos-westus3-dev-httpsettings",
+                                "ngsa-java-westus3-dev.cse.ms~ngsa-java-westus3-dev-httpsettings",
+                                "ngsa-memory-westus3-dev.cse.ms~ngsa-memory-westus3-dev-httpsettings",
+                                "loderunner-westus3-dev.cse.ms~loderunner-westus3-dev-httpsettings"
```
[the remark this answers is not in the document]
Comment (TravisFallsMSFT) [reply monitoring/alerts/metricsAlerts/dev/asb-dev-westus2-AppEndpointDown.json]: I have adjusted this file, and did a search for cse.ms and changed all other references to it to the austinrdc.dev.  I have also adjusted all the guids associated.

File: monitoring/alerts/metricsAlerts/pre/asb-pre-centralus-AppEndpointDown.json (hunk 2 above)
Comment (TravisFallsMSFT) [reply monitoring/alerts/metricsAlerts/pre/asb-pre-centralus-AppEndpointDown.json]: fixed.  thank you.

File: monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json (hunk 4 above)
Comment (TravisFallsMSFT) [reply monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json]: fixed.  thank you.

File: monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json (hunk 3 above)
Comment (TravisFallsMSFT) [reply monitoring/alerts/metricsAlerts/pre/asb-pre-westus2-AppEndpointDown.json]: fixed.  thank you.

**Review by AAkindele — APPROVED** [review]
🚀

**Pull request merged**